client/src/theme.js

```diff
@@ -1,58 +1,67 @@
 import { createTheme } from "@mui/material";
 
-// TODO: Add brand colors map
-// eslint-disable-next-line
-const BRAND_COLORS = {
+const COLORS = {
+  HYPERLINK_BLUE: "#005EA2",
+  BODY_TEXT: "#2F3032",
+  LIGHT_GREY_TEXT: "#78787A",
+  BACKGROUND: "#FAF0E7",
+  CARD_BG: "#FFFFFF",
+  NAVBAR_BG: "#F2DAC4",
   BLUEBERRY: "#021944",
   LYCHEE: "#F6A794",
-  MATCHA: "#B1D5B2",
-  MILK_TEA: "#F2DAC4",
-  TARO: "#E4CFF7",
+  CARD_CHIPS: "#ABBFE3",
+  ERROR_BUTTON: "#D21C1C",
+  ERROR_BG: "#F8D7D7",
+  SLIDEOUT_NOTES_BG: "#FFF5ED",
   THAI_TEA: "#E0A878",
+  MILK_TEA: "#F2DAC4",
+  LOGO_BROWN: "#98513A",
+  OFFWHITE_GREY: "#D9D9D9",
+  PAGINATION_BG: "#F9E5DC",
+  SEARCHBAR_ICON_BORDER: "#DBAE9C",
+  THAI_FILL_INACTIVE: "#B4AAA0",
+  BOBA_POSITION: "#A9917B",
 };
 
 const theme = createTheme({
   palette: {
-    brown: {
-      50: "#FAF0E7",
-      100: BRAND_COLORS.MILK_TEA,
-      200: "#E0A878",
-      300: "#A4857B",
-      400: "#98513A",
-      500: "#683323",
+    primary: {
+      main: COLORS.LYCHEE,
+      light: COLORS.PAGINATION_BG,
+      dark: COLORS.SEARCHBAR_ICON_BORDER,
+    },
+    secondary: {
+      main: COLORS.BLUEBERRY,
+      light: COLORS.CARD_CHIPS,
+      dark: COLORS.HYPERLINK_BLUE,
     },
-    gray: {
-      50: "#FFFFFF",
-      100: "#F7EDEA",
-      200: "#E4E2E0",
-      300: "#8C8C8C",
-      400: "#E7D8CE",
-      500: "#D9D9D9",
-      600: "#B4AAA0",
-      700: "#979797",
-      800: "#2F3032",
-      900: "#000000",
```

Comment:
nice, yeah the createTheme shoudl give us a ton of generated colors based on some of the base values we pass in, have we compared what those output looks like yet? does just adding `grey: main: somecolor` spit out all of the grey['50-900'] values as we expect?

Reply:
I will take a look and compare the colors - is this something I should discuss with the design team as well? Since the colors generated by createTheme might not be the same as the ones used in design

Reply:
Nope! just dont think we need to get so granular we override each and every shade and tone of the colors we use.

Design should drive the core color palette, and we can use auto generated light/darks if we need, or if design demands, we can override them, but in terms of naming and clarity, everything should make sense. colors are colors, not components. components have colors.

So we establish the base, and then apply the decided colors to the components as we see fit